src/tests/createHeadlessForm.test.js

```diff
@@ -1810,6 +1810,22 @@ describe('createHeadlessForm', () => {
     });
   });
 
+  describe('when a field is text', () => {
```

Comment:
Hi Jakob! I'm not understanding, is this a bug report or a PR with the actual fix that you forgot to push? 😅

Reply:
I can open an issue for it, it's just a small bug, but I don't have much time to fix it right now, just pushed the WIP state for now.

It will probably stay as draft for a couple more weeks, you should have access to `jguddas:fix/crashing-when-format-is-uri` if you want to fix this quick.

Reply:
I don't have time either, it's okay to stay here for a couple of months ;)